docs/user/dashboard/lens-percentage.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[analyze-data-as-percentage-in-line]]
+== Advanced guide: Analyze data as percentage in Lens
+
+It is often useful to with with percentages within your data: from computing a single metric about performance goal, to representing the proportion of metrics over time.
+
+The aim of this guide is to provide an overview of representing percentages with several visualizations within the Lens editor: some of the content of this guide is inspired by this blog post based on TSVB, but applied to the Lens editor.
```

Comment:
I don't think this is needed. We can show users how they can apply percentages, and since not all users are familiar with our blogs, I don't wan to create confusion.